components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -215,6 +216,8 @@ class RewardsServiceImpl : public RewardsService,
   const RewardsNotificationService::RewardsNotificationsMap&
   GetAllNotifications() override;
 
+  void SetContributionAmount(double amount) const override;
```

Comment:
`double amount` should be changed to `const double amount`

Reply:
done